guide/10-mapping-and-visualization-alt/part7_utility_functions_for_the_map_widget.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<h2>Show message in the map widget via <code>display_message</code></h2><p> </p><p> </p><p><strong>Note:</strong> Only one message can be sent at a time, and multiple messages will not show up. Let's experiment with the first example by changing the <code>print</code> method to <code>display_message</code> so the desired text output can be shown inside the widget. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>